src/tutorial/microbe_editor/MicrobeEditorTutorialGUI.cs

```diff
@@ -413,7 +413,7 @@ public void HandleShowingATPBarHighlight()
             throw new InvalidOperationException("Balance bar highlight control is not set");
 
         bool eitherVisible = atpBalanceIntroduction.Visible || negativeAtpBalanceTutorial.Visible;
-        AtpBalanceBarHighlight.Visible = atpBalanceIntroduction.Visible;
+        AtpBalanceBarHighlight.Visible = eitherVisible;
```

Comment:
I specifically made this change as I found it to not be super nice to blank out the editor multiple times to show the location of the bars. Is this actually better? This doesn't let the player cross reference other information in the editor about their not enough ATP situation.

Reply:
Do you mind if I revert this one change so that all the other good parts can be merged?

Reply:
Go right ahead